jobs/jobs-operator/jobs-operator/src/main.py

```diff
@@ -0,0 +1,66 @@
+import kopf
+from kubernetes import client as kclient
+import os
+import logging
+import sys
+
+logging.basicConfig(
+    format="%(asctime)s [%(levelname)s] %(name)s: %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+    stream=sys.stdout,
+)
+logger = logging.getLogger(__name__)
+
+
+@kopf.on.create("curibio.dev", "v1", "jobrunners")  # TODO look at how to change the on create to prod.net
+def create_fn(body, spec, **kwargs):
+    # Get info from grafana object
+    namespace = body["metadata"]["namespace"]
+    job_queue = spec["job_queue"]
+    qp_name = f"{job_queue}-queue-processor"
+    logger.info(f"Starting {qp_name} in namespace {namespace}")
+
+    POSTGRES_PASSWORD = kclient.V1EnvVar(name="POSTGRES_PASSWORD", value=os.getenv("POSTGRES_PASSWORD"))
+    QUEUE_VAR = kclient.V1EnvVar(name="QUEUE", value=job_queue)
+    ECR_REPO = kclient.V1EnvVar(name="ECR_REPO", value=spec["ecr_repo"])
```

Comment:
can these values come through the spec fields in the CRD?

Reply:
The spec fields in the CRD get passed here to the operator but I don't think they can get passed directly into the queue processor without being added to the container like below

Reply:
oh, I see, I misunderstood what was happening here